src/nginxconfig/templates/setup_sections/ssl.vue

```diff
@@ -32,7 +32,7 @@ THE SOFTWARE.
                     <span v-html="$t('templates.setupSections.ssl.generateDiffieHellmanKeysByRunningThisCommandOnYourServer')"></span>
                     <br />
                     <BashPrism :key="`${$props.data.global.nginx.nginxConfigDirectory.computed}-${diffieHellmanValue}`"
-                               :cmd="`openssl dhparam -out ${$props.data.global.nginx.nginxConfigDirectory.computed}/dhparam.pem ${diffieHellmanValue}`"
+                               :cmd="`curl https://ssl-config.mozilla.org/ffdhe2048.txt > ${$props.data.global.nginx.nginxConfigDirectory.computed}/dhparam.pem`"
```

Comment:
Maybe 2048 should be replaced with `${diffieHellmanValue}`. There are three files available in https://wiki.mozilla.org/Security/Archive/Server_Side_TLS_4.0#Pre-defined_DHE_groups (2048, 3072 and 4096).

Reply:
Agree, `diffieHellmanValue` should be used still -- I'd also suggest using wget here instead, as more base systems tend to have it available than curl.

Reply:
It's also dynamic (old, intermediate, modern), so the 1024 should be generated by `openssl`, but intermediate should be the pre-generated `dhparam` from the RFC. I'll update this PR when I have some more time.